packages/syntax-core/src/TapArea/TapArea.module.css

```diff
@@ -0,0 +1,59 @@
+.tapArea {
+  box-sizing: border-box;
+}
+
+.fullWidth {
+  width: 100%;
+}
+
+.disabled {
+  filter: opacity(50%);
+  background-image: none;
+  transform: none;
+  cursor: auto;
+}
+
+.enabled {
+  cursor: pointer;
+}
+
+.enabled:hover {
```

Comment:
Not entirely sure if this is true, but I think I saw a few of joey's designs and not all of them necessarily needed a hover state on all of the clickable cards. 

Maybe worth considering to have hover styles be a prop that can be toggled off (like disabled)

Reply:
@somethiiing We can make the default to have hover styles for now - if we do need it, we can always add a prop to disable them. However, in this case, I would lean to having more consistency across our UX / codebase